content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NodeJS Compatibility Requirements 
+kind: documentation
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 50
+---
+
+## Language and framework compatibility
+
+### Supported Node.js versions
+
+The Datadog Node.js library supports the following Node.js versions:
```

Comment:
Replace with--

The Datadog Node.js library supports Node 14 and higher. In general, we follow the Node.js LTS policy. For more information, please refer to the [APM documentation](https://docs.datadoghq.com/tracing/trace_collection/compatibility/nodejs/#nodejs-version-support)